__tests__/app.test.js

```diff
@@ -63,3 +63,75 @@ describe.only("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET : /api/articles object including total comments", () => {
+  test("/api/articlezz -> route that does not exist: 404 Not Found", () => {
+    return request(app)
+      .get("/api/articlezz")
+      .expect(404)
+      .then(({ body: { message } }) => {
+        expect(message).toBe("not found");
+      });
+  });
+
+  test("200 : returns object with all major fields sort by descending DEFAULT", () => {
+    const expectedData = [
+      {
+        article_id: 3,
+        title: "Eight pug gifs that remind me of mitch",
+        topic: "mitch",
+        author: "icellusedkars",
+        body: "some gifs",
+        created_at: "2020-11-03T09:12:00.000Z",
+        votes: 0,
+        comment_count: 2,
+      },
```

Comment:
why are you only expecting 5 articles to be returned here?

There are 12 in your test data!

The articles that are missing are ones that have no comments...